libs/components/action-bars/src/lib/modules/summary-action-bar/summary-action-bar.component.scss

```diff
@@ -122,6 +122,11 @@
   );
 }
 
+.sky-summary-action-bar-errors {
+  max-height: 100px;
```

Comment:
Why the magic number here?

Reply:
this is based off of how modal handles there being a ton of errors based off of this thread from the modal footer error pr https://github.com/blackbaud/skyux/pull/1484#issuecomment-1631415210

Reply:
from flex conversation: these values matched how modal handles it and a custom class was created to be used in both locations.